Codegen/src/Nodes/FunctionDeclarationCG.cs

```diff
@@ -68,6 +76,16 @@ internal class FunctionDeclarationCG : Compiler.Nodes.FunctionDeclaration, Gener
                         workingStorageSection.AddRange(toAddRange, 0);
                         GenerateCodeToCallPublicProc(originalNode, pdiv,  workingStorageSection, linkageSection);
                     }
+                    else if (OriginalNode.IsFlagSet(Node.Flag.UseGlobalStorage))
+                    {
+                        Node[] toAddRange =
```

Comment:
This code seems a duplicate of line 202 `Node[] globalCalltoAddRange =`
Plus, line 202 there is comment line that is missing here.

Can you factorize?